client/state/data-layer/wpcom/comments/index.js

```diff
@@ -36,6 +63,38 @@ export const fetchPostComments = ( { dispatch, getState }, action ) => {
 	}, action ) );
 };
 
+export const writePostComment = ( { dispatch }, action ) => {
+	const { siteId, postId, parentCommentId, commentText } = action;
+	const placeholder = createPlaceholderComment( commentText, postId, parentCommentId );
+
+	// Insert a placeholder
+	dispatch( {
+		type: COMMENTS_RECEIVE,
+		siteId,
+		postId,
+		comments: [ placeholder ],
+		skipSort: !! parentCommentId
+	} );
+
+	const path = !! parentCommentId
+		? `/sites/${ siteId }/comments/${ parentCommentId }/replies/new`
+		: `/sites/${ siteId }/posts/${ postId }/replies/new`;
```

Comment:
I have no problem on changing the naming on this. I did that on #1353. But I'm not entirely convinced `resources` is the best name.
I've implemented this on dfcb45f437d903375c55426dad13e509143c82fa and it's really confusing.

At the moment we have `state/data-layer/wpcom/comments` and `state/data-layer/wpcom/sites/comments` and comments can depend on posts, pages (if enabled) and other comments. And we'll need handlers for likes, status changes and edits. Why don't we use the much simpler `state/data-layer/wpcom/comments/new` (and `comments/likes`, `comments/status`) instead of trying to force this to that convention?
We don't really need to map the endpoint structure in such a convoluted way to make it "findable" ("accessible"?)